tflib/imagetest/helm/main.tf

```diff
@@ -26,22 +33,30 @@ variable "values" {
   default     = {}
 }
 
+variable "values_file" {
+  description = "Path to values file on the local filesystem."
+  type        = string
+  default     = ""
+}
+
 variable "timeout" {
   description = "The timeout on the helm install."
+  type        = string
   default     = "5m"
 }
 
 locals {
   install_cmd = <<-EOa
 apk add helm
 helm install ${var.name} ${var.chart} \
-  --namespace ${var.namespace} --create-namespace \
-  --repo ${var.repo} \
-  --wait --wait-for-jobs \
-  --timeout ${var.timeout} \
+  %{if var.namespace != "default"}--namespace ${var.namespace} --create-namespace%{endif} \
```

Comment:
what does this buy us over just passing in `var.namespace`? which is always going to be at _least_ `default`?

Reply:
Meh I was going for succinctness for the inputs but in reality it doesn't actually change anything. Good call, it doesn't buy us anything